Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -173,12 +173,12 @@ def get_modified_files_for_testing(files_string):
                 continue
 
             elif any(file in file_path for file in (PACKS_PACK_META_FILE_NAME, PACKS_WHITELIST_FILE_NAME)):
-                pack_sample_tests.append(file_path)
+                pack_tests.append(file_path)
```

Comment:
actually, my bad.
this list doesn't contain the pack tests so the name is not correct.
also, you can save the pack name here so the set is smaller.